### ISSUE TITLE
can't plot absorption coefficient, transmission, absorption,cross sections

### ISSUE THREAD

**Issue by fsubobe** [hitranonline/hapiest#49]
I'm using osx 14.6 with Python 3.7.7 that includes pip 20.2.2 that includes numpy 1.19.0 . I am able to plot bands, but when I attempt to plot the other quantities I receive the following 

    File "/usr/local/lib/python3.7/site-packages/numpy/core/function_base.py", line 113, in linspace
    |       num = operator.index(num)
    |   
    |   'numpy.float64' object cannot be interpreted as an integer

Error executing work request:  'numpy.float64' object cannot be interpreted as an integer <class 'TypeError'> None

A similar error occurs using HAPI.

Its difficult to use python 3.6 as pip for that version is not compatible with openssl 1.1.  Going back to  openssl 1.0 causes other issues.

Thus, is there a way to modify hapiest to get around this issue?

**Comment by jkarns275** [comment]
Hi, this was a known bug in hapi that has been fixed by Roman in subsequent versions of hapi. It was caused by a change in numpy I believe. I just pushed a fix 036bdda720629c89d13964ebda579a2945026ed5 (the fix is updating the version of hapi used). Let me know if this solves the issue.

**Comment by fsubobe** [comment]
Thanks. I played around with numbly a bit more and found that it worked if I downgraded numpy. I don’t like to do such changes to standard packages, however.

Thanks again.

Bob Ellingson

On Jun 29, 2020, at 12:17 PM, Josh Karns <notifications@github.com> wrote:

﻿

Hi, this was a known bug in hapi that has been fixed by Roman in subsequent versions of hapi. It was caused by a change in numpy I believe. I just pushed a fix 036bdda<https://urldefense.com/v3/__https://github.com/hitranonline/hapiest/commit/036bdda720629c89d13964ebda579a2945026ed5__;!!PhOWcWs!gulOhMNKopr_343Fc5yXWAX9i0zXz_WldTAdETDwbMbg0o0mIsFCkFvfZ9dVYZOf$> (the fix is updating the version of hapi used). Let me know if this solves the issue.
